src/articulation.ts

```diff
@@ -143,13 +160,24 @@ const getInitialOffset = (note, position) => {
       return 0;
     }
   }
-};
+}
 
 export class Articulation extends Modifier {
-  static get CATEGORY() {
+  note?: Note;
+  readonly type: string;
+
+  protected render_options: { font_scale: number };
+  protected articulation?: ArticulationStruct;
+  // glyph defined calling reset in constructor
+  // eslint-disable-next-line
+  protected glyph!: Glyph;
+  static DEBUG: boolean;
+
+  static get CATEGORY(): string {
     return 'articulations';
   }
-  static get INITIAL_OFFSET() {
+
+  static get INITIAL_OFFSET(): number {
```

Comment:
Can you make this a static property instead?

Reply:
done